training-react/src/app/components/User/layout.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import withLoadingScreen from '../LoadingScreen';
+
+import styles from './styles.scss';
+
+const UserDetails = ({ user, userSection, editUser }) => (
```

Comment:
As a standard we follow, functional components should be defined as regular `function`, not arrow ones.